### ISSUE TITLE
Terminal links: Understand line and column information after a file path

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#21182]
While testing #21021.

tsc generates error messages of the following form:
```
[10:13:10] Error: /Users/me/Development/projects/vscode/extensions/html/server/src/modes/htmlMode.ts(36,31): Property 'findDocumentSymbols' does not exist on type 'LanguageService'.
[10:13:10] Error: /Users/me/Development/projects/vscode/extensions/markdown/src/extension.ts(50,36): Property 'com_mands' does not exist on type 'typeof 'vscode''.
```

The files are correctly linkified.  We should have a link detection extension that also includes the line and column information so that we can directly jump to the error by clicking the link.

**Comment by Tyriar** [comment]
Related: https://github.com/Microsoft/vscode/issues/20676 https://github.com/Microsoft/vscode/issues/15179

**Comment by jonaskello** [comment]
I think this is solved in the April 2017 (version 1.12) [release](https://code.visualstudio.com/updates/v1_12#_integrated-terminal-link-line-and-column-ranges).

However I would like it to match also when the line and column are within brackets like `[10, 22]` in order to work with tslint and other tools that use this style. Related tslint issue [here](https://github.com/palantir/tslint/issues/2722).

**Comment by Tyriar** [comment]
Thanks for pointing that out @jonaskello, closing as duplicate. Moved the bracket issue to https://github.com/Microsoft/vscode/issues/26330
